code/frameworks/nextjs/src/images/next-image-stub.tsx

```diff
@@ -1,32 +1,67 @@
 /* eslint-disable no-underscore-dangle, @typescript-eslint/no-non-null-assertion */
 import * as React from 'react';
 import type * as _NextImage from 'next/image';
+import type * as _NextLegacyImage from 'next/legacy/image';
```

Comment:
is this going to be fine for projects that do not have such path? e.g. Next 11

Reply:
yep. See the webpack `IgnorePlugin` implementation